build/run_container.sh

```diff
@@ -49,19 +49,28 @@ run_build_container() {
       cmd=(/bin/bash)
   fi
 
+  # In case of `minikube`, kube config stores full path to a certificates,
+  # thus the simples way to get working minikube in the build container is
+  # to bind original path to minikube settings to the container.
+  local path_to_minikube_dir="${HOME}/.minikube"
```

Comment:
```suggestion
  local minikube_dir_path="${HOME}/.minikube"
```

Reply:
I am not able to understand the use of minikube while starting the build container. Can you please describe that?
`run_build_container` should just be able to run a command on a container right? Or does it also need to know how the kubernetes is setup.
If it needs to know how Kubernetes is setup, how does it work for kind?

Reply:
In case of `minikube` in `kubeconfig` stored full paths to the `minio`'s certificates. It's mentioned in https://github.com/kanisterio/kanister/pull/2378:

> Error: Kubernetes cluster unreachable: invalid configuration: [unable to read client-cert /Users/sergey.aksenov/.minikube/profiles/minikube/client.crt for minikube due to open /Users/sergey.aksenov/.minikube/profiles/minikube/client.crt: no such file or directory, unable to read client-key /Users/sergey.aksenov/.minikube/profiles/minikube/client.key for minikube due to open /Users/sergey.aksenov/.minikube/profiles/minikube/client.key: no such file or directory, unable to read certificate-authority /Users/sergey.aksenov/.minikube/ca.crt for minikube due to open /Users/sergey.aksenov/.minikube/ca.crt: no such file or directory]

and in comment above this variable.

Reply:
does this mean the tests were not running on minkube earlier they were just supported on kind clusters? And after this change they would also run on minikube?

Reply:
Currently if you'll try to run some command within container, which will try to run another container, you'll get `ERROR: Cannot connect to the Docker daemon at unix:///path_to_socket. Is the docker daemon running?` error, it has been fixed in second commit of this PR. Exactly because of binding `~/.docker` directory.

Reply:
Exactly :)

Reply:
>  local minikube_dir_path="${HOME}/.minikube"

fixed